plugin/fzf.vim

```diff
@@ -262,7 +266,7 @@ try
     set shell=sh
   endif
 
-  if has('nvim') && len(filter(range(1, bufnr('$')), 'bufname(v:val) =~# ";#FZF"'))
+  if has('nvim') && len(filter(range(1, bufnr('$')), 'bufname(v:val) =~# ";#FZF"')) || exists('s:curr_job')
```

Comment:
I think we should also check the return value of `job_status(s:curr_job)`. Or Vim will keep complaining once we kill the tmux pane running FZF.